docs/releasing/README.md

```diff
@@ -135,18 +135,19 @@ export P_VERSION=v0.7.4
     git checkout master
     git checkout -b "prep-$P_VERSION-release"
     ```
-2. Update the [RELEASES.md](../../RELEASES.md) file with the next pending release version, `$P_VERSION`, for maintainers to place their changes as they make them. 
-3. Modify the [plugin/evm/version.go](../../plugin/evm/version.go) `Version` global string variable and set it to `$P_VERSION`.
-4. Create a pull request (PR) from your branch targeting master, for example using [`gh`](https://cli.github.com/):
+2. Bump the version number to the next pending release version, `$P_VERSION`
+  - Update the [RELEASES.md](../../RELEASES.md) file with `$P_VERSION`, creating a space for maintainers to place their changes as they make them. 
+  - Modify the [plugin/evm/version.go](../../plugin/evm/version.go) `Version` global string variable and set it to `$P_VERSION`.
+3. Create a pull request (PR) from your branch targeting master, for example using [`gh`](https://cli.github.com/):
     ```bash
     gh pr create --repo github.com/ava-labs/coreth --base master --title "chore: prep next release $P_VERSION"
     ```
-5. Wait for the PR checks to pass with
+4. Wait for the PR checks to pass with
```

Comment:
nit: I think the markdown supports auto-ordering if you keep using `1.` for all them. No worries for now, but something can be used in the next time.